packages/syft/src/syft/service/network/network_service.py

```diff
@@ -615,8 +664,7 @@ def delete_route_on_peer(
         self,
         context: AuthedServiceContext,
         peer: NodePeer,
-        route: NodeRoute | None = None,
-        route_id: UID | None = None,
```

Comment:
Deprecated updation or Deletion of node routes using route id, since route id was not unique and consistent during creation of node routes.